### PR TITLE
Add "set/activate open window" feature

### PR DESCRIPTION
**Pull request by AndreasHassing**
Also allows resetting the "open window" state back.

To check if a window is open in a zone, call
`IsOpenWindowDetected()` on a `State` object.
You need to add a `using` of `KoenZomers.Tado.Api.Helpers` to see the
extension method.

### CODE REVIEW

**Comment by AndreasHassing** [comment]
Finally got Tado to detect an open window.

File: DotNetStandardApi/Helpers/StateExtensions.cs [hunk 1]
```diff
@@ -0,0 +1,10 @@
+﻿using KoenZomers.Tado.Api.Entities;
```
Comment (KoenZomers) [inline DotNetStandardApi/Helpers/StateExtensions.cs:1]: Personally don't see the added value of adding this extension. Why not just use the bool from the State object directly?

Comment (AndreasHassing) [reply DotNetStandardApi/Helpers/StateExtensions.cs:1]: I prefer the extension method as it lets me ignore the fact that it's a `Nullable<bool>`.

But I can just declare it in my own project(s) 😊.

Comment (KoenZomers) [reply DotNetStandardApi/Helpers/StateExtensions.cs:1]: I'd say just use GetValueOrDefault if you don't want it to be a nullable bool :) If you don't mind, I will implement it without the extension to keep it in line with how the rest of the code has been done already. Thanks for this nice contribution nevertheless!

**Pull request merged**